Update Azure windows environment

https://dev.azure.com/openlawlibrary/TAF/_build/results?buildId=977&view=results

```
##[error]This is a scheduled windows-2016 brownout. 
The windows-2016 environment is deprecated and will be removed on April 1st, 2022. 
For more details, see https://github.com/actions/virtual-environments/issues/5238
,##[error]The remote provider was unable to process the request.
```

--- a/azure-pipelines.yml
+++ b/azure-pipelines.yml
@@ -21,12 +21,12 @@ strategy:
       imageName: "macOS-10.15"
       pythonVersion: "3.6"
     windows-64bit-py36:
-      imageName: "vs2017-win2016"
+      imageName: "windows-2019"
       pythonVersion: "3.6"
       platform: x64
       winArch: "win64"
     windows-32bit-py36:
-      imageName: "vs2017-win2016"
+      imageName: "windows-2019"
       pythonVersion: "3.6"
       platform: x86
       winArch: "win32"
@@ -38,12 +38,12 @@ strategy:
       imageName: "macOS-10.15"
       pythonVersion: "3.8"
     windows-64bit-py38:
-      imageName: "vs2017-win2016"
+      imageName: "windows-2022"
       pythonVersion: "3.8"
       platform: x64
       winArch: "win64"
     windows-32bit-py38:
-      imageName: "vs2017-win2016"
+      imageName: "windows-2022"
       pythonVersion: "3.8"
       platform: x86
       winArch: "win32"

